tests/utils/shopper.js

```diff
@@ -294,7 +294,7 @@ export const shopper = {
 				expect( page ).toMatch(
 					customerShippingDetails.lastname
 				),
-				expect( page ).toMatch( customerShippingDetails.company),
+				// expect( page ).toMatch( customerShippingDetails.company),
```

Comment:
Should we ping Rubik team about these changes?

Reply:
Yes, I've pinged them in Slack and added them as reviewers as well, I'll wait for them before merging.